opentelemetry-exporter-gcp-monitoring/tests/test_normalize_label_key.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from opentelemetry.exporter.cloud_monitoring import _normalize_label_key
+
+
+@pytest.mark.parametrize(
+    ("key", "expected"),
+    [
+        ("valid_key_1", "valid_key_1"),
+        ("hellø", "hellø"),
```

Comment:
not sure if this expected, but the Go impl does do the same thing https://go.dev/play/p/OJbkDHqwote

the GCM docs just say "letter"

Reply:
Should be possible to integration test pretty easily, right?

Reply:
Ya in the collector repo.